tests/test_obo_reading.py

```diff
@@ -142,3 +142,12 @@ def test_presence_of_obsolete_nodes() -> None:
     assert "BTO:0000311" in nodes
     node = nodes["BTO:0000311"]
     assert node["is_obsolete"] == "true"
+
+
+def test_curly_braces_in_synonym() -> None:
+    """Test that we can handle curly braces inside synonyms"""
+    line = 'synonym: "10*3.{copies}/mL" EXACT [] {http://purl.obolibrary.org/something="AB"}'
+    tag, value, trailing_modifier, comment = parse_tag_line(line)
+    assert tag == "synonym"
+    assert value == '"10*3.{copies}/mL" EXACT []'
+    assert trailing_modifier
```

Comment:
Can we rename this test to `test_parse_tag_line_curly_braces` and move it up 2 tests so it is located with the other  `test_parse_tag_*` functions?

Happy to do this for you if it is not convenient.

Reply:
Sure, done in latest commit